packages/unmock-node/src/backend/index.ts

```diff
@@ -1,8 +1,10 @@
-import fr from "follow-redirects";
```

Comment:
Happy to be rid of this ;) Should `follow-redirects` be removed from the dependencies then?

Reply:
Yup!